src/Illuminate/Support/Str.php

```diff
@@ -68,6 +68,19 @@ public static function of($string)
         return new Stringable($string);
     }
 
+    /**
+     * Get an acronym consisting of the first letter of each word in a given string.
+     */
+    public static function acronym(string $string): string
+    {
+        $acronym = '';
+        foreach (explode(' ', $string) as $word) {
+            $acronym .= $word[0];
```

Comment:
A warning will be raised here if you pass a string with multiple spaces. `Hello  World` for instance

Reply:
Should this account for newlines/return carriages?

Should all punctuation be stripped? `Hello World! ! !` would return `HW!!`

Reply:
I've updated the method and added additional test cases for this.

Reply:
I've changed it to `/[^a-zA-Z]+/`  which excludes any non-alphabetic characters.